### PR TITLE
Improve wikipedia expando

### PR DESCRIPTION
**Pull request by thybag**
* Remove Wikipedia attribution
* make in-comment text info look better (ie. Wikipedia expando's)

fixes https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues/2860 https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues/2859

Screeny of wiki text-expando in comments with new styles:
wiki-text-screen

### CODE REVIEW

File: lib/css/modules/_showImages.scss [hunk 1]
```diff
@@ -129,3 +129,11 @@ body:not(.res-showImages-displayImageCaptions) {
 		display: none !important;
 	}
 }
+
+// style expando text content in comments
+.commentarea div.usertext-body.madeVisible .md {
+	background-color: #fafafa;
```
Comment (andytuba) [inline lib/css/modules/_showImages.scss]: Darkmode too

File: lib/modules/hosts/wikipedia.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 addLibrary('mediaHosts', 'wikipedia', {
 	domains: ['wikipedia.org'],
-	logo: 'https://en.wikipedia.org/favicon.ico',
+	attribution: false,
```
Comment (andytuba) [inline lib/modules/hosts/wikipedia.js]: thanks

**Comment by thybag** [comment]
Now with nightmode styles :)

**Comment by andytuba** [comment]
:+1:

**Pull request merged**